src/plugins/mer/meremulatormodedialog.h

```diff
@@ -30,6 +30,7 @@
 #ifndef MEREMULATORMODEDIALOG_H
 #define MEREMULATORMODEDIALOG_H
 
+#include <QAction>
```

Comment:
This should go to the .cpp file unless I missed something.

Reply:
Thanks for the quick review. I'll reflect the comment.